db/migrate/20220527201132_add_author_and_company_to_articles.rb

```diff
@@ -0,0 +1,6 @@
+class AddAuthorAndCompanyToArticles < ActiveRecord::Migration[6.1]
+  def change
+    add_reference :articles, :user, foreign_key: true
```

Comment:
Do we need to add null false constraint?

Reply:
Mikhail and I discussed this. We have articles in the prod that do not have an author. As part of this PR, we want to migrate it, assign an author to these articles and add a constraint next time.